docs/stable/3-waku-usage.md

```diff
@@ -1,89 +1,98 @@
-This allows allows anyone to listen on a single channel.
+the case of public chats, the channel name maps directly to its Waku topic.
+This allows anyone to listen on a single channel.
 
-Additionally, since anyone can receive Whisper envelopes, it relies on the
+Additionally, since anyone can receive Waku envelopes, it relies on the
 ability to decrypt messages to decide who is the correct recipient. We do
 however not rely on this property, but instead implement another secure
-transport layer on top of Whisper.
+transport layer on top of Waku.
 
 Finally, we use an extension of Whisper to provide the ability to do offline
 messaging.
 
+```js
+// TODO is the extension for offline messaging part of Waku? Or do we still use a Whisper extension?
+```
+
 ## Reason
 
 Provide routing, metadata protection, topic-based multicasting and basic
 encryption properties to support asynchronous chat.
 
 ## Terminology
 
-* *Whisper node*: an Ethereum node with Whisper V6 enabled (in the case of geth, it's `--shh` option)
-* *Whisper network*: a group of Whisper nodes connected together through the internet connection and forming a graph
-* *Message*: decrypted Whisper message
+* *Waku node*: an Ethereum node with Waku V1 enabled
+* *Waku network*: a group of Waku nodes connected together through the internet connection and forming a graph
+* *Message*: decrypted Waku message
 * *Offline message*: an archived envelope
 * *Envelope*: encrypted message with metadata like topic and Time-To-Live
 
-## Whisper packets
-
-| Packet Name | Code | EIP-627 | References |
-| --- | --: | --- | --- |
-| Status | 0 | ✔ | [Handshake](#handshake) |
-| Messages | 1 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
-| PoW Requirement | 2 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
-| Bloom Filter | 3 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
-| Batch Ack | 11 | 𝘅 | Undocumented |
-| Message Response | 12 | 𝘅 | Undocumented |
-| P2P Sync Request | 123 | 𝘅 | Undocumented |
-| P2P Sync Response | 124 | 𝘅 | Undocumented |
-| P2P Request Complete | 125 | 𝘅 | [4/WHISPER-MAILSERVER](https://specs.status.im/spec/4) |
-| P2P Request | 126 | ✔ | [4/WHISPER-MAILSERVER](https://specs.status.im/spec/4) |
-| P2P Messages | 127 | ✔/𝘅 (EIP-627 supports only single envelope in a packet) | [4/WHISPER-MAILSERVER](https://specs.status.im/spec/4) |
-
+## Waku packets
+
+| Packet Name          | Code | References |
+| -------------------- | ---: | --- |
+| Status               |    0 | [Handshake](#handshake), [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#status) |
+| Messages             |    1 | [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#messages) |
+| PoW Requirement      |    2 | [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#pow-requirement-update) |
```

Comment:
2,3, 123 & 124 are not part of Waku spec, and thus not used afaik.
See updates here: https://github.com/vacp2p/specs/pull/119 (and if they get accepted / corrected)

Reply:
Yeah. This section needs a rewrite after our chat yesterday https://discordapp.com/channels/624205794384281629/626744455361331222/707997524631355464